### PR TITLE
Add APIClarity to the Learning section

### PR DESCRIPTION
**Pull request by akpsgit**

### CODE REVIEW

**Comment by philsturgeon** [comment]
Woah this is really cool! Are you involved with the project or are you a user? 

This says its only OpenAPI v2.0, which makes it not particularly useful for the majority of OpenAPI users as OAS3.0 has been out for years and OAS3.1 is out now too.

We do not add 2.0-only tools to the site anymore, as there are 3.x tools that can handle things like this.

Could you could provide a unique use case for why this should be put on the site?

**Comment by akpsgit** [comment]
Hi @philsturgeon, yes I'm involved with the project :) Actually, we were focused on v2.0 only because we wrote it in Golang, for CNCF environments & workloads (were Golang is popular as well). Currently, there is no official support in goswagger for v3.0, so we started from v2.0, but our next thing will be to support v3.0 and v3.1 (upcoming weeks). So should be really useful detecting APIs of Golang workloads and CNCF environments right away, but it's the most urgent action item on our roadmap to support the latest versions, and additional standards like gRPC, GraphQL, etc.

**Comment by philsturgeon** [comment]
Brilliant! If you have a link to an issue that talks about v3 support, I think we have a v3_link: yaml option you can pop it in. We'll add it if you show you're on the case. :)

**Comment by akpsgit** [comment]
Cool :) 
The task was opened and assigned: https://github.com/apiclarity/apiclarity/issues/39
I found an example with the v3_progress_link option, should I use this one or v3_link as suggested?
Also, here is the roadmap presentation from the CNCF webinar mentioning v3 as top priority :)
https://youtu.be/2vYdWf5hjNU?t=2022 
Thanks!

File: _data/tools.yml [hunk 1]
```diff
@@ -564,6 +564,16 @@
...
+- name: APIClarity
+  category: learning
+  language: Golang, JavaScript
+  link: https://apiclarity.io/
+  github: https://github.com/apiclarity/apiclarity
+  description:
+    Reconstruct Open API Specifications from real-time workload traffic seamlessly.
+  v2: true
+  v3: false
+
```
Comment (philsturgeon) [inline _data/tools.yml:576]:
```suggestion
v3_link: https://github.com/apiclarity/apiclarity/issues/39

```

**Comment by philsturgeon** [comment]
Thank you!

**Pull request merged**